src/test/java/com/hubspot/jinjava/lib/filter/DefaultFilterTest.java

```diff
@@ -0,0 +1,38 @@
+package com.hubspot.jinjava.lib.filter;
+
+import com.hubspot.jinjava.Jinjava;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.HashMap;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Created by manishdevgan on 25/06/19.
+ */
+public class DefaultFilterTest {
+    Jinjava jinjava;
+
+    @Before
+    public void setup() {
+        jinjava = new Jinjava();
+        jinjava.getGlobalContext().registerClasses(DefaultFilter.class);
+    }
+
+    @Test
+    public void itSetsDefaultStringValues() {
+        assertThat(jinjava.render("{% set d=d | default(\"some random value\") %}{{ d }}", new HashMap<>())).isEqualTo("some random value");
```

Comment:
Could you either leave these as long lines or hard-wrap them on a logical boundary, like `,` (preferred)?